libs/ml/include/ml/serializers/ml_types.hpp

```diff
@@ -3731,5 +3737,38 @@ struct MapSerializer<ml::optimisers::LearningRateParam<T>, D>
   }
 };
 
+/**
```

Comment:
it might be worth to create a separate file for this routine (because ml_types is already 3740 lines)